tests/composer.json

```diff
@@ -6,6 +6,6 @@
         }
     ],
     "require": {
-        "buttercms/api-wrapper": "dev-master"
+        "buttercms/buttercms-php": "dev-master"
```

Comment:
It updates the tests to match the current version of the client code. `buttercms/api-wrapper` is the old name for the composer package. It was changed to `buttercms/buttercms-php` here:

https://github.com/ButterCMS/buttercms-php/commit/7a43590f932a2902298b194a1e8156cb69346182#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780L2